pkg/indexmanagement/scripts.go

```diff
@@ -34,8 +34,16 @@ CMD=$(cat <<END
 from __future__ import print_function
 import json,sys
 r=json.load(sys.stdin)
+if not isinstance(r,dict):
+  print("Unable to determine write indices. Response was not a json payload")
+  print(r)
```

Comment:
can we change this to be a single print line instead?
`print("Unable to determine write indices. Response was not a json payload: \n{response}".format(response=r))`